### PR TITLE
Add support for custom timestamp name and field to nginx parser

### PR DESCRIPTION
**Pull request by nathanleclaire**
ping @maplebed

### CODE REVIEW

File: httime/httimetest/fake_nower.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	"reflect"
+	"time"
+)
+
+type FakeNower struct {
+	FakeNow time.Time
+}
+
+func (f *FakeNower) Now() time.Time {
+	if !reflect.DeepEqual(f.FakeNow, time.Time{}) {
```
Comment (maplebed) [inline httime/httimetest/fake_nower.go]: how about `if !f.FakeNow.IsZero()`?

Comment (nathanleclaire) [inline httime/httimetest/fake_nower.go]: :+1:

File: parsers/htjson/htjson.go [hunk 2]
```diff
@@ -4,53 +4,33 @@ package htjson
...
-}
-
 type Options struct {
-	TimeFieldName string `long:"timefield" description:"Name of the field that contains a timestamp"`
-	Format        string `long:"format" description:"Format of the timestamp found in timefield (supports strftime and Golang time formats)"`
+	TimeFieldName   string `long:"timefield" description:"Name of the field that contains a timestamp"`
+	TimeFieldFormat string `long:"format" description:"Format of the timestamp found in timefield (supports strftime and Golang time formats)"`
 
-	NumParsers int `hidden:"true" description:"number of mongo parsers to spin up"`
+	NumParsers int `hidden:"true" description:"number of htjson parsers to spin up"`
```
Comment (emfree) [inline parsers/htjson/htjson.go]: :facepalm:

Comment (nathanleclaire) [inline parsers/htjson/htjson.go]: it happens... best to clean it up and not make same mistake in future ;P

**Comment by nathanleclaire** [comment]
>  Huge thanks for cleaning all this up :) 

You're welcome!

**Pull request merged**